courses/machine_learning/deepdive2/structured/labs/4c_keras_wide_and_deep_babyweight.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>It looks like this block is missing some things that don't have corresponding TODOs?</p><p> </p><p>Either way, would be good to have additional TODOs in this block, including the code concerned with creating the wide and deep model. For example, put TODOs to create 'dnn_inputs' and 'wide_inputs' (....which seem to be missing from here?) and TODO to create 'model', etc.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Thanks for catching that, bad cross over from the DNN lab, I think from fixing extra white space of all things.</p>